src/views/Hearing/Sidebar.js

```diff
@@ -35,6 +37,38 @@ class Sidebar extends React.Component {
     );
   }
 
+  getLanguageChanger() {
+    const {hearing, activeLanguage} = this.props;
+    const availableLanguages = { fi: 'Kuuleminen Suomeksi', sv: 'Enkäten på svenska', en: 'Questionnaire in English'};
+    const languageOptionsArray = config.languages.map((lang) => {
+      if (getAttr(hearing.title, lang, {exact: true}) && lang === activeLanguage) {
```

Comment:
And a yet another bug. Don't know if this has been here since the beginning of the language branch, but the links at the top bar do not work outside the main page. The user is stuck in whichever subpage they are.